CONTRIBUTING.md

```diff
@@ -1,42 +1,59 @@
 # Contributing to Catalyst
+
 Thanks for showing interest in contributing!
 
-The following is a set of guidelines for contributing to Catalyst. These are just guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+The following is a set of guidelines for contributing to Catalyst. These are just guidelines, not rules.
+Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## Repository Structure
+
+Catalyst is a monorepo that contains the code for the Catalyst Next.js application inside of `core/`,
+and supporting packages such as the GraphQL API client and the `create-catalyst` CLI in `packages/`.
+
+The default branch for this repository is called `canary`. This is the primary development branch where
+active development takes place, including the introduction of new features, bug fixes, and other changes
+before they are released in stable versions.
```

Comment:
This might be a good area to explain the different tags we release, and their practical usage (e.g., explain which Makeswift tag is deployed by OCC? Will need help from someone more familiar with this area.